src/test/resources/projects/server/build.gradle

```diff
@@ -46,3 +46,4 @@ dependencies {
 }
 
 /*endpoints-plugin-hostname*/
+/*endpoints-plugin-basePath*/
```

Comment:
Ditto comment on all lowercase.

Reply:
oh you found it.

Reply:
yeah it's just a random string replacement, it can be whatever we want.

Reply:
Okay thanks that makes sense.

Reply:
but `basePath` is fine. :)